### ISSUE TITLE
PayPal Instant Payment Notification (IPN) certificate changes

### ISSUE THREAD

**Issue by ace-dent** [wp-e-commerce/WP-e-Commerce#1994]
No sure whether this is an issue yet... From PayPal:
" PayPal is upgrading the certificate for www.paypal.com to SHA-256. This endpoint is also used by merchants using the Instant Payment Notification (IPN) product. This upgrade is scheduled for 30/9/2015; however, we may need to change this date on short notice to you to align to the industry security standard. ... Full technical details can be found in our [Merchant Security System Upgrade Guide ](https://www.paypal-knowledge.com/resources/sites/PAYPAL/content/live/FAQ/1000/FAQ1766/en_US/2015%20Merchant%20Security%20System%20Upgrade%20Guide%20%28U.K.%20English%29.pdf). In addition, our [2015-2016 SSL Certificate Change](https://www.paypal-knowledge.com/infocenter/index?page=content&id=FAQ1766&expand=true&locale=en_US) microsite contains a schedule of our service upgrade plan."

**Comment by JustinSainton** [comment]
See this post: https://wpecommerce.org/news/wp-ecommerce-and-the-latest-paypal-sha-256-announcement/

tl;dr: we're all good here.
